src/main/java/com/FoodyBuddy/Model/Buyer.java

```diff
@@ -0,0 +1,210 @@
+package com.FoodyBuddy.Model;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import org.hibernate.envers.Audited;
+
+@Entity
+@Audited
+@Table(name = "Buyer")
+public class Buyer{
+	
+	@Id
+	@Column(name = "id", nullable = false)
+	@GeneratedValue
+	private int id;
+	
+	@Column(name = "apartment_id", nullable = false)
+	@ManyToOne
```

Comment:
All of us have done the mistake here . near @ManyToOne annottation @column shouldn't be specified

The code should be: